lib/telegram/parse.js

```diff
@@ -21,4 +21,9 @@ module.exports = function(messageObject) {
       type: 'telegram'
     };
   }
+
+  return {
+    originalRequest: messageObject,
+    type: 'telegram'
+  };
```

Comment:
I think we should still have sender and text, just to keep the standard format. It's ok if text is empty, we already have that for Slack [here](https://github.com/skylinezum/claudia-bot-builder/blob/master/lib/slack/parse.js#L15).